kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/dsl/intern/KoverAggregatedExtensionImpl.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.aggregation.settings.dsl.intern
+
+import org.gradle.api.provider.SetProperty
+
+internal abstract class KoverAggregatedExtensionImpl {
```

Comment:
What's the purpose of this class? It seems to be unused

Reply:
Yes, it's useless now.